src/modules/launcher/Plugins/Microsoft.Plugin.WindowWalker/Components/ResultHelper.cs

```diff
@@ -97,6 +97,11 @@ private static string GetSubtitle(Window window)
                 subtitleText += $" - {Resources.wox_plugin_windowwalker_Desktop}: {window.Desktop.Name}";
             }
 
+            if (!window.Process.Responding)
+            {
+                subtitleText = $"{Resources.wox_plugin_windowwalker_Process}: {window.Process.Name} [{Resources.wox_plugin_windowwalker_NotResponding}]";
+            }
```

Comment:
```suggestion
            if (!window.Process.Responding)
            {
                subtitleText += $" [{Resources.wox_plugin_windowwalker_NotResponding}] ";
            }
```

And switch with line 95-98.

Reply:
Fixed in commit [32a3f56](https://github.com/microsoft/PowerToys/pull/33167/commits/32a3f568843af0160cf598d324c48a14c0d954ce)